terraform_fortios_provider/provider_prerender

```diff
@@ -0,0 +1,301 @@
+  visibility = "enable"
+}
+
+#VPN IPsec phase1-interface
+resource "fortios_vpnipsec_phase1interface" "FortiDEMOphase1" {
+  name = "FortiDEMO"
+  interface = "port1"
+  peertype = "any"
+  net_device = "enable"
+  proposal = "aes128-sha256 aes256-sha256 aes128-sha1 aes256-sha1"
+  wizard_type = "static-fortigate"
+  remote_gw = "${forti_demo_ip}"
+  psksecret = "fortidemo"
+}
+
+#VPN IPsec phase2-interface
+resource "fortios_vpnipsec_phase2interface" "FortiDEMOphase2" {
+  name = "FortiDEMO"
+  phase1name = fortios_vpnipsec_phase1interface.FortiDEMOphase1.name
+  proposal = "aes128-sha1 aes256-sha1 aes128-sha256 aes256-sha256 aes128gcm aes256gcm chacha20poly1305"
+  src_addr_type = "name"
+  dst_addr_type = "name"
+  src_name = "FortiDEMO_local"
+  dst_name = "FortiDEMO_remote"
+
+  depends_on = [
+    fortios_vpnipsec_phase1interface.FortiDEMOphase1,
+    fortios_firewall_address.local,
+    fortios_firewall_address.remote,
+    ]
+ }
+
+#VPN Firewall Policies
+resource "fortios_firewall_policy" "vpnpolicy1" {
+  policyid = 100
+  name = "vpn_FortiDEMO_local"
+  action = "accept"
+  nat = "enable"
+  schedule = "always"
+
+  srcintf {
+    name = "port1"
+  }
+
+  dstintf {
+    name = "FortiDEMO"
+  }
+
+  srcaddr {
+    name = "FortiDEMO_local"
+  }
+
+  dstaddr {
+    name = "FortiDEMO_remote"
+  }
+
+  service {
+    name = "ALL"
+  }
+
+  depends_on = [
+    fortios_system_interface.FortiDEMOinterface
+    ]
+}
+
+resource "fortios_firewall_policy" "vpnpolicy2" {
+  policyid = 101
+  name = "vpn_FortiDEMO_remote"
+  action = "accept"
+  nat = "enable"
+  schedule = "always"
+
+  srcintf {
+    name = "FortiDEMO"
+  }
+
+  dstintf {
+    name = "port1"
+  }
+
+  srcaddr {
+    name = "FortiDEMO_remote"
+  }
+
+  dstaddr {
+    name = "FortiDEMO_local"
+  }
+
+  service {
+    name = "ALL"
+  }
+
+  depends_on = [
+    fortios_system_interface.FortiDEMOinterface
+    ]
+}
+
+#Firewall Policies
+resource "fortios_firewall_policy" "policy1" {
+  policyid = 103
+  name = "port1 to port1"
+  action = "accept"
+  nat = "enable"
+  schedule = "always"
+
+  srcintf {
+    name = "port1"
+  }
+
+  dstintf {
+    name = "port1"
+  }
+
+  srcaddr {
+    name = "all"
+  }
+
+  dstaddr {
+    name = "all"
+  }
+
+  service {
+    name = "ALL"
+  }
+}
+
+#VPN Interface
+resource "fortios_system_interface" "FortiDEMOinterface" {
+  name          = fortios_vpnipsec_phase2interface.FortiDEMOphase2.name
+  description   = "VPN Interface"
+  type          = "tunnel"
+  vdom          = "root"
+  allowaccess   = "ping ssh fabric"
+  ip            = "0.0.0.0 0.0.0.0"
+  remote_ip     = "0.0.0.0 0.0.0.0"
+  tcp_mss       = 1400
+  autogenerated = "auto"
+}
+
+#Router Configuration
+resource "fortios_router_static" "Access" {
+  dst = "0.0.0.0 0.0.0.0"
+  device = "FortiDEMO"
+  dstaddr = "FortiDEMO_remote"
+  seq_num = 1
+
+  depends_on = [
+    fortios_system_interface.FortiDEMOinterface
+    ]
+}
+
+resource "fortios_router_static" "blackhole" {
```

Comment:
Might want to add these resources that have no variable interpolation into your fortios.tf.